weekly-updates/weekly-update.2015-09-11.md

```diff
@@ -0,0 +1,71 @@
+* **timers**: Improved timer performance from porting the 0.12 implementation, plus minor fixes (Jeremiah Senkpiel) [#2540](https://github.com/nodejs/node/pull/2540), (Julien Gilli) [nodejs/node-v0.x-archive#8751](https://github.com/nodejs/node-v0.x-archive/pull/8751) [nodejs/node-v0.x-archive#8905](https://github.com/nodejs/node-v0.x-archive/pull/8905)
+
+### Known issues
+
+See https://github.com/nodejs/node/labels/confirmed-bug for complete and current list of known issues.
+
+* Some uses of computed object shorthand properties are not handled correctly by the current version of V8. e.g. `[{ [prop]: val }]` evaluates to `[{}]`. [#2507](https://github.com/nodejs/node/issues/2507)
+* Some problems with unreferenced timers running during `beforeExit` are still to be resolved. See [#1264](https://github.com/nodejs/node/issues/1264).
+* Surrogate pair in REPL can freeze terminal. [#690](https://github.com/nodejs/node/issues/690)
+* Calling `dns.setServers()` while a DNS query is in progress can cause the process to crash on a failed assertion. [#894](https://github.com/nodejs/node/issues/894)
+* `url.resolve` may transfer the auth portion of the url when resolving between two full hosts, see [#1435](https://github.com/nodejs/node/issues/1435). 
+
+### Node.js Foundation Combines Node.js and io.js Into Single Codebase in New Release
+
+Node.js Foundation posts an entry about Node.js new release and introduction of Node Foundation. [The detail is here](https://nodejs.org/en/blog/announcements/foundation-v4-announce/)
+
+### Node.js Interactive
+
+[Node.js Interactive](http://events.linuxfoundation.org/events/node-interactive) is a new, annual, vendor-neutral conference for Node.js.  It is being led by the newly formed Node.js Foundation.
+
+### Community Updates
+
+* We *still* need a V8 maintainer for our LTS build! Head on over [to GitHub](https://github.com/nodejs/LTS/issues/28) to see if the requirements match your capabilities.
+* According to [Raygun blog](https://raygun.io/blog/2015/09/nodejs-and-io-js-are-now-one/), Node.js v4.0.0 is 8% faster than Node.js v0.12.
+* Tony Pujals, evangelism WG member, posts an [entry about Node.js v4.0.0](https://www.linkedin.com/pulse/node-v400-here-tony-pujals), He introduced about ES6 on Node.js v4.0.0, LTS, and SemVer.
+
+If you have spotted or written something about Node.js and io.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [Node.js Italian Conference](http://nodejsconf.it/) tickets are on sale, October 10th at Desenzano - Brescia, Italy
+* [EmpireNode](http://2015.empirenode.org/), October 23rd at New York, US.
+* [JSConf CO](http://www.jsconf.co/), October 16th - 17th at Ruta N, Medellin, Columbia
```

Comment:
It looks like the EmpireNode and JSConf CO event entries are reversed. JSConf CO is happening first so it should be before EmpireNode in the list.